python/paddle/nn/functional/norm.py

```diff
@@ -403,3 +402,118 @@ def instance_norm(x,
+
+        In the above equation:
+
+        - :math:`size` : The number of channels to sum over.
+        - :math:`k` : The offset (avoid being divided by 0).
+        - :math:`\\alpha` : The scaling parameter.
+        - :math:`\\beta` : The exponent parameter.
+
+
+        Args:
+            x (Tensor): The input 3-D/4-D/5-D tensor. The data type is float32.
+            size (int): The number of channels to sum over.
+            alpha (float, optional): The scaling parameter, positive. Default:1e-4
+            beta (float, optional): The exponent, positive. Default:0.75
+            k (float, optional): An offset, positive. Default: 1.0
+            data_format (str, optional): Specify the data format of the input, and the data format of the output
+                will be consistent with that of the input. An optional string from:
+                If x is 3-D Tensor, the string could be `"NCL"` or `"NLC"` . When it is `"NCL"`,
+                the data is stored in the order of: `[batch_size, input_channels, feature_length]`.
+                If x is 4-D Tensor, the string could be  `"NCHW"`, `"NHWC"`. When it is `"NCHW"`,
+                the data is stored in the order of: `[batch_size, input_channels, input_height, input_width]`.
+                If x is 5-D Tensor, the string could be  `"NCDHW"`, `"NDHWC"` . When it is `"NCDHW"`,
+                the data is stored in the order of: `[batch_size, input_channels, input_depth, input_height, input_width]`.
+            name (str, optional): Name for the operation (optional, default is None). For more information,
+                please refer to :ref:`api_guide_Name`.
+
+        Returns:
+            A tensor storing the transformation result with the same shape and data type as input.
+
+
+        Examples:
+
+        .. code-block:: python
+
+            import paddle
+
+            x = paddle.rand(shape=(3, 3, 112, 112), dtype="float32")
+            y = paddle.nn.functional.local_response_norm(x, size=5)
+            print(y.shape)  # [3, 3, 112, 112]
+        """
+    if not in_dygraph_mode():
+        check_variable_and_dtype(x, 'x', ['float32'], 'local_response_norm')
+    if data_format not in ['NCL', 'NLC', 'NCHW', 'NHWC', 'NCDHW', 'NDHWC']:
+        raise ValueError(
+            "Attr(data_format) of Op(local_response_norm) got wrong value: received "
+            + data_format +
+            " but only 'NCL'/'NLC'/'NCHW'/'NHWC'/'NCDHW'/'NDHWC' supported.")
+
+    sizes = x.shape
+    dim = len(sizes)
+    if dim < 3:
+        raise ValueError(
+            'Expected 3D or higher dimensionality input, but got {} dimensions'.
+            format(dim))
+
+    channel_last = True if data_format[-1] == "C" else False
+
+    div = fluid.layers.unsqueeze(paddle.multiply(x, x), axes=[1])
+    if channel_last:
+        if dim == 3:
```

Comment:
this part is a bit too repetitive, could use some consolidation.

Reply:
Done